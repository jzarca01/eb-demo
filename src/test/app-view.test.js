// Link.react.test.js
import React from 'react';
import AppView from '../components/app-view';
import renderer from 'react-test-renderer';
import {
    StaticRouter
} from 'react-router-dom';

it('renders without even crashing', () => {
  const component = renderer.create(
    <AppView />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});