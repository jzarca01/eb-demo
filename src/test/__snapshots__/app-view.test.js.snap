// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="ui segment pushable"
>
  <div
    className="ui inverted vertical labeled icon ui push left thin visible sidebar menu"
  />
  <div
    className="pusher"
  >
    <div
      className="ui basic segment"
    >
      <div>
        <p>
          Hello Home
        </p>
      </div>
    </div>
  </div>
</div>
`;
